provisioning/service/samples/readme.md

```diff
@@ -0,0 +1,78 @@
+# Samples for the Azure IoT Device Provisioning Service SDK for Node.js
+
+This file can be found in http://github.com/azure/azure-iot-sdk-node/provisioning/service/samples
+
+This folder contains simple samples showing how to use the various features of the Microsoft Azure IoT Device Provisioning Service from an application running JavaScript code.
+
+## List of samples
+
+* Create a simple IndividualEnrollment object.
+   *  [create_individual_enrollment.js][create-individual-enrollment]
+* Create a simple EnrollmentGroup object.
+   *  [create_enrollment_group.js][create-enrollment-group]
+* Create a couple of IndividualEnrollments and delete them.
+   *  [create_delete.js][create-delete]
+* Create queries for IndividualEnrollments, EnrollmentGroups and DeviceRegistrationStates that belong to an EnrollmentGroup.
+   *  [query.js][query-link]
+* Perform BulkEnrollmentOperations to do various CRUD operations.
+   *  [run_bulk_operation.js][run-bulk-operation]
+
+
+## How to run the samples
+
+In order to run the device samples you will first need the following prerequisites:
+* Node.js v4 or above on your target device. (Check out [Nodejs.org](https://nodejs.org/) for more info)
+* [Setup Azure IoT Device Provisioning ][lnk-setup-iot-provisioning] Stop before executing the the Create and provision a simulated device section.  The samples below will perform these steps.
+
+<<<<<<<<<<<<<<<<<<link to how to obtain an endorsment key!!!!>>>>>>>>>>>>>>>>>>
```

Comment:
This should be how to run the simulated TPM and extract the EK from it, yes?

Reply:
Something like that.

---
In reply to: [150296512](https://github.com/Azure/azure-iot-sdk-node/pull/142#discussion_r150296512) [](ancestors = 150296512)